pkg/proxy/iptables/proxier.go

```diff
@@ -190,14 +189,6 @@ func NewProxier(ipt utiliptables.Interface, exec utilexec.Interface, syncPeriod
 		return nil, fmt.Errorf("can't set sysctl %s: %v", sysctlRouteLocalnet, err)
```

Comment:
is this sysctl call still appropriate for the proxier to set, rather than the network plugin?